chromadb/utils/embedding_functions.py

```diff
@@ -643,6 +645,53 @@ def __call__(self, input: Union[Documents, Images]) -> Embeddings:
         return embeddings
 
 
+class RoboflowEmbeddingFunction(EmbeddingFunction[Images]):
+    def __init__(
+        self, api_key: str, api_url = "https://infer.roboflow.com"
+    ) -> None:
+        self._api_url = api_url
+        self._api_key = api_key
+
+    def __call__(self, images: Images = [], prompt: str = None) -> Embeddings:
```

Comment:
Chroma's `EmbeddingFunction` protocol provides a unified interface for `__call__` methods, please see https://github.com/chroma-core/chroma/blob/86b749ad42457c34044274a30a8ac0c6527d8469/chromadb/utils/embedding_functions.py#L636

For things to work properly, there has to be exactly and only an `input` argument of the `EmbeddingFunction`'s type. Switching should happen on the type of the argument.